build/ci/job-template.yml

```diff
@@ -27,7 +27,7 @@ jobs:
       nightlyBuildRunPath: $(Build.SourcesDirectory)/bin/AnyCPU.$(_configuration)/Microsoft.ML.NightlyBuild.Tests/$(_targetFramework)
       packageUpdaterProjPath: $(Build.SourcesDirectory)/test/Microsoft.ML.NugetPackageVersionUpdater/Microsoft.ML.NugetPackageVersionUpdater.csproj
       versionFilePath: $(Build.SourcesDirectory)/test/Microsoft.ML.NugetPackageVersionUpdater/latest_versions.txt
-      PROCDUMP_PATH: '$(Build.SourcesDirectory)/Tools/procdump/'
+      PROCDUMP_PATH: '$(Build.SourcesDirectory)/Tools/Procdump_Files/'
```

Comment:
Naming nit: What is wrong with just procdump? Why Procdump_Files? (A directory will always contain files or other directories :-) )